extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Delivery Feedback Events",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Contains all delivery event fields.",
+  "definitions": {
+    "feedbackevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/feedbackType": {
+          "title": "Feedback Type",
+          "type": "string",
+          "description": "The type of message event."
+        },
+        "https://ns.adobe.com/experience/campaign/retryCount": {
+          "title": "Retry Count",
+          "type": "integer",
+          "description": "Number of failed attempts before the current event."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery": {
+          "title": "The campaign activity originating a message to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Delivery ID",
+              "type": "string",
+              "description": "The campaign activity originating this message."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Id",
+              "type": "string",
+              "description": "Campaign of which this delivery was a part."
+            },
+            "xdm:from": {
+              "title": "From",
+              "type": "string",
+              "description": "Address used as a from/sender/caller address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+            },
+            "xdm:type": {
+              "title": "Delivery Type",
+              "type": "string",
+              "description": "Mode of delivery for the messages sent."
+            },
+            "xdm:label": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description": "A human-friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:name": {
+              "title": "Delivery Internal Name",
+              "type": "string",
+              "description": "A human-friendly identifier of the campaign activity which is originating this message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message": {
+          "title": "A single message sent to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Message Identifier",
+              "type": "string",
+              "description": "Identifier of the message."
+            },
+            "xdm:subject": {
+              "title": "Message Subject",
+              "type": "string",
+              "description": "Subject line of the message."
+            },
+            "xdm:size": {
```

Comment:
Same here! xdm:emailMessageSize should be more appropriate, else work on the object definition /email/message on the line no 64.

Reply:
Same as above.